src/seedu/addressbook/ui/TextUi.java

```diff
@@ -111,9 +111,7 @@ public void showInitFailedMessage() {
 
     /** Shows message(s) to the user */
     public void showToUser(String... message) {
-        for (String m : message) {
-            out.println(LINE_PREFIX + m.replace("\n", LS + LINE_PREFIX));
-        }
+            out.println(Formatter.concat(message));
```

Comment:
1. perhaps you can move more formatting related stuff (like the divider, offset etc.,) to the formatter; think about it!
2. however, you are doing this part right; you are passing the message to the formatter and getting back the formatted string to print :+1: